lib/index.js

```diff
@@ -125,22 +125,22 @@ FSTree.prototype.calculatePatch = function(otherFSTree, isEqual) {
       } else {
         // pre-cleanup file removals should occure in-order, this ensures file
         // -> directory transforms work correctly
-        operations.push(command);
+        additions.push(command);
       }
 
-      // remove operations
+      // remove additions
     } else if (x.relativePath > y.relativePath) {
       // theirs
       j++;
-      operations.push(addCommand(y));
+      additions.push(addCommand(y));
     } else {
       if (!isEqual(x, y)) {
         command = updateCommand(y);
 
         if (x.isDirectory()) {
           removals.push(command);
         } else {
-          operations.push(command);
+          additions.push(command);
```

Comment:
I think this branch is actually an `update` not a addition. So the naming is a-bit funky.